gnark/README.md

```diff
@@ -0,0 +1,42 @@
+# Sample Gnark Circuits
+
+In this directory, we provide a handful of Gnark circuits which have already been prepared to use with Forge. You may find them useful as a reference when writing your own circuits; or you may use these as circuit uploads if you are just beginning to try out Forge.
+
+All of the tar files are compressed versions of the corresponding folder. Rather than uploading the entire directory to Forge, you will upload it's compressed .tar.gz file. In order to request a proof for one of these circuits, consult the input.json file in the corresponding folder.
+
+It is allowed (but unneccesary) to include extraneous files in your compressed directory such as readme's, example inputs, and unused code. However, you should not include any executables.
+
+See Sindri's GitBook for more complete instructions.
+
+## Available Circuits
+
+### v0.9.1
+
+| Name | Size (DEGREE) | Original Source | Functionality | 
+| ---- | ---- | --------------- | ------------- | 
+| cubic | 3 | [Gnark Github Examples](https://github.com/Consensys/gnark/blob/master/examples/cubic/cubic.go) | Checks $x^3 + x + 5 == y$ for public $y$ and private $x$|
+| poseidon | 214 | [Vocdoni's Gnark Primitives](https://github.com/vocdoni/gnark-crypto-primitives/tree/main) | Computes the Poseidon hash of a preimage  |
+
+
+## Circuits Requirements
+- Upload all your go language source code for a circuit
+  - include your module definition file `go.mod`
+  - our current prover configuration uses gnark `v0.8.0` and gnark-crypto `v0.9.1`. We cannot gaurantee compatibility with circuits relying on newer features.
```

Comment:
As per Arya, should we be including the version for gnark-crypto?

s/gaurantee/guarantee/

Reply:
Spellcheck is addressed by https://github.com/Sindri-Labs/forge-sample-data/pull/11/commits/7ed8b2d90f4abae75309c6fa490966011fdc2f08

I see your point - that we do not technically need to list the gnark-crypto version and should instead just state the frontend version.  However the current circuit type says `v0.9.1` and this repo should match the current Forge version.  So I'd prefer to keep it in for now.  However, I'll make a ticket for updating Forge based on what we learned and then we can revisit these docs.

Reply:
thanks for the explanation, let's revisit a version change across the board, maybe next time they have a new fe release? this wfm right now